src/nemos/basis.py

```diff
@@ -255,7 +482,6 @@ def __init__(
         self.n_basis_funcs = n_basis_funcs
         self._n_input_dimensionality = 0
         self._check_n_basis_min()
-        self._conv_args = args
```

Comment:
why is this removed? (seems unrelated to this PR)

Reply:
this is removed because variable positional arguments are not allowed for BaseEstimators.